src/Pages/Components/Header.jsx

```diff
@@ -1,13 +1,44 @@
-export default function Header() {
+import { useState } from "react";
+
+export default function Header({ projects, setProjects }) {
+  const [searchInput, setSearchInput] = useState("");
+  console.log("searchinput", searchInput.replace(/\s+/g, "-").toLowerCase());
```

Comment:
Might be an idea to remove this console log if you are finished with it :)

Reply:
okey will do